event_sync/mod_event_sync_component.lua

```diff
@@ -187,6 +187,16 @@ function EventData:get_occupant_array()
     return output;
 end
 
+--- Returns number of active occupants
+function EventData:get_active_occupants()
```

Comment:
Can we name the function something like `get_active_occupant_count` to make it obvious that it returns a number instead of occupant data like the other functions?

Reply:
Done.